docs/events.md

```diff
@@ -1,6 +1,16 @@
 Events
 ======
 
+The Box API supports two types of event streams -- one for the events specific to a particular user
+ and one for all of the events in an enterprise.
+
+* [User Events](#user-events)
+* [Enterprise Events](#enterprise-events)
+
+
+User Events
+===========
```

Comment:
The heading levels on these need to be adjusted; they should be second-level headings (under top-level "Events") using the `--------` underline.